docs/source/onnxruntime/modeling_ort.mdx

```diff
@@ -0,0 +1,112 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Optimum Inference with Onnxrunime
+
+Optimum Inference is a utility package for building and running inference with accleratored runtime like Onnxruntime. 
+Optimum Inference can be used to load optimized models from the [Hugging Face Hub](hf.co/models) and create pipelines 
+to run acclerated inference without rewriting your APIs.
+
+## Switching from Transfromers to Optimum Inference
+
+The Optimum Inference models are API compatible with Transformers models, meaning you can just replace your `AutoFor*` with the optimum `OnnxFor*` class. 
```

Comment:
Question: should we use `OnnxModelForXxx` instead of `OnnxForXxx`? Or if we only use ONNX Runtime, should it be `ORTModelForXxx`? In `transformers` we use `AutoModelForXxx` so adopting a similar convention might make sense

```suggestion
The Optimum Inference models are API compatible with Hugging Face Transformers models. This means you can just replace your `AutoModelForXxx` class with the corresponding `OnnxForXxx` class in `optimum`. For example, this is how you can use a question answering model in `optimum`: 
```